charts/database/templates/database-deployment.yaml

```diff
@@ -39,6 +39,8 @@ spec:
           env:
             - name: DATABASE_STORAGE
               value: "{{.Values.global.storage}}"
+            - name: PGCTLTIMEOUT
+              value: "{{.Values.postgres.timeout}}"
```

Comment:
the ability to modify how we package the chart and relevant changes in the repo in one PR is awesome. 👌